ClearKeep/Scenes/InCall/InCallView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  InCallView.swift
+//  ClearKeep
+//
+//  Created by HOANDHTB on 20/07/2022.
+//
+
+import SwiftUI
+import ChatSecure
+import CommonUI
+
+struct InCallView: View {
+	@ObservedObject var viewModel: CallViewModel
+	
+	var body: some View {
+		Group {
+			if viewModel.callGroup {
+				GroupIncallView(viewModel: viewModel)
+			} else {
+				PeerCallView(viewModel: viewModel)
+			}
+		}
+		.grandientBackground()
+		.edgesIgnoringSafeArea(.all)
+		.onAppear(perform: {
+			if let callBox = CallManager.shared.calls.first {
+				viewModel.updateCallBox(callBox: callBox)
+			}
+		})
+	}
+	
+	enum AlertCall {
+		case changeTypeCall
+		case endCall
+		case error(error: LoginViewError)
+		
+		var title: String {
+			if case .error(let err) = self {
+				return err.title
+			}
+			
+			return "Call.Warning".localized
+		}
+		
+		var description: String {
+			switch self {
+			case .changeTypeCall:
+				return "Call.SwitchCall".localized
+			case .endCall:
+				return"Call.EndQuestion".localized
+			case .error(let err):
+				return err.message
+			}
+		}
+		
+		var primaryButtonTitle: String {
+			switch self {
+			case .changeTypeCall:
+				return "Call.OK".localized
+			case .endCall:
+				return "Call.Leave".localized
+			case .error(let error):
+				return error.primaryButtonTitle
+			}
+		}
+		
+		var secondButtonTitle: String {
+			return "Call.Cancel".localized
+		}
+	}
+}
+
+struct InCallView_Previews: PreviewProvider {
+    static var previews: some View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/InCall/InCallView.swift#L74"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>InCallView.swift:74</code></td>
    </tr>
  </tbody>
</table>